### DISCUSSION TITLE
如何排除若干个字段?

### DISCUSSION THREAD

**Discussion by any35** [rbatis/rbatis#287] [General]
如何排除若干个字段呢?

**Comment by zhuxiujia** [comment]
> 如何排除若干个字段呢?

可以使用#[crud_table( table_name:"biz_activity" |  table_columns:"id,name,delete_flag")]
来限定使用哪几个字段。

参考 https://rbatis.github.io/rbatis.io/#/?id=%e8%a1%a8%e6%a8%a1%e5%9e%8b%e5%ae%9a%e4%b9%89

**Comment by any35** [comment]
多谢回复, 文档已经很仔细看了, 包括每次版本升级都还要去看源代码. 希望有增加排除部分字段的功能.
 
1. 实际工作中有些共性需求, 比如DDD, 聚合根里面的事件列表是不希望保存到数据库的.
2. 但字段比较多这样容易出错, 两边同时维护也大大增加了心智负担.
3. 排除是另外一个方面的含义, 语义更明确一些.
